Wave1_data/1.preprocess_data/scripts/1.define_platemap.py

```diff
@@ -0,0 +1,182 @@
+    "H05": "LPS 1 ug/ml + Nigericin 3uM",
+    "I05": "LPS 1 ug/ml + Nigericin 1 uM",
+    "J05": "LPS 1 ug/ml + Nigericin 0.5uM",
+    "K05": "LPS 1 ug/ml + Nigericin 0.1 uM",
+    "L05": "Ab1-42 10 uM",
+    "M05": "Ab1-42 2 uM",
+    "N05": "Ab1-42 0.4 uM",
+    "C06": "CTL (complete medium only)",
+    "D06": "LPS 10 ug/ml",
+    "E06": "LPS 1 ug/ml",
+    "F06": "LPS 0.1 ug/ml",
+    "G06": "LPS 1 ug/ml + ATP 2.5 mM",
+    "H06": "Flagellin 10 ug/ml",
+    "I06": "Flagellin 1 ug/ml",
+    "J06": "Flagellin 0.1 ug/ml",
+    "K06": "H2O2 500 uM",
+    "L06": "H2O2 100 uM",
+    "M06": "H2O2 100 nM",
+    "N06": "no Hoechst",
+    "C07": "DMSO CTL",
+    "D07": "Thapsigargin 10 uM",
+    "E07": "Thapsigargin 1 uM",
+    "F07": "Thapsigargin 0.5uM",
+    "G07": "LPS 1 ug/ml + Nigericin 5uM",
+    "H07": "LPS 1 ug/ml + Nigericin 3uM",
+    "I07": "LPS 1 ug/ml + Nigericin 1 uM",
+    "J07": "LPS 1 ug/ml + Nigericin 0.5uM",
+    "K07": "LPS 1 ug/ml + Nigericin 0.1 uM",
+    "L07": "Ab1-42 10 uM",
+    "M07": "Ab1-42 2 uM",
+    "N07": "Ab1-42 0.4 uM",
+    "C08": "CTL (complete medium only)",
+    "D08": "LPS 10 ug/ml",
+    "E08": "LPS 1 ug/ml",
+    "F08": "LPS 0.1 ug/ml",
+    "G08": "LPS 1 ug/ml + ATP 2.5 mM",
+    "H08": "Flagellin 10 ug/ml",
+    "I08": "Flagellin 1 ug/ml",
+    "J08": "Flagellin 0.1 ug/ml",
+    "K08": "H2O2 500 uM",
+    "L08": "H2O2 100 uM",
+    "M08": "H2O2 100 nM",
+    "N08": "no Hoechst",
+    "C09": "DMSO CTL",
+    "D09": "Thapsigargin 10 uM",
+    "E09": "Thapsigargin 1 uM",
+    "F09": "Thapsigargin 0.5uM",
+    "G09": "LPS 1 ug/ml + Nigericin 5uM",
+    "H09": "LPS 1 ug/ml + Nigericin 3uM",
+    "I09": "LPS 1 ug/ml + Nigericin 1 uM",
+    "J09": "LPS 1 ug/ml + Nigericin 0.5uM",
+    "K09": "LPS 1 ug/ml + Nigericin 0.1 uM",
+    "L09": "Ab1-42 10 uM",
+    "M09": "Ab1-42 2 uM",
+    "N09": "Ab1-42 0.4 uM",
+    "C10": "CTL (complete medium only)",
+    "D10": "LPS 10 ug/ml",
+    "E10": "LPS 1 ug/ml",
+    "F10": "LPS 0.1 ug/ml",
+    "G10": "LPS 1 ug/ml + ATP 2.5 mM",
+    "H10": "Flagellin 10 ug/ml",
+    "I10": "Flagellin 1 ug/ml",
+    "J10": "Flagellin 0.1 ug/ml",
+    "K10": "H2O2 500 uM",
+    "L10": "H2O2 100 uM",
+    "M10": "H2O2 100 nM",
+    "N10": "no Hoechst",
+    "C11": "DMSO CTL",
+    "D11": "Thapsigargin 10 uM",
+    "E11": "Thapsigargin 1 uM",
+    "F11": "Thapsigargin 0.5uM",
+    "G11": "LPS 1 ug/ml + Nigericin 5uM",
+    "H11": "LPS 1 ug/ml + Nigericin 3uM",
+    "I11": "LPS 1 ug/ml + Nigericin 1 uM",
+    "J11": "LPS 1 ug/ml + Nigericin 0.5uM",
+    "K11": "LPS 1 ug/ml + Nigericin 0.1 uM",
+    "L11": "Ab1-42 10 uM",
+    "M11": "Ab1-42 2 uM",
+    "N11": "Ab1-42 0.4 uM",
+}
+platemap_df = pd.DataFrame.from_dict(
+    dict_platemap, orient="index", columns=["treatment"]
+)
+platemap_df.reset_index(inplace=True)
+platemap_df.rename(columns={"index": "well"}, inplace=True)
+platemap_df["treatment"] = platemap_df["treatment"].str.replace(
+    "CTL (complete medium only)", "Media"
+)
```

Comment:
Why include this replacement here when you could change it in the dictionary above?

Reply:
True, changing now.